<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionclient</class>
 <widget class="QMainWindow" name="gestionclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1259</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>1081</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);


font: 87 8pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter un client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1071</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-image: url(:/img/images (1).jpg);</string>
      </property>
      <property name="title">
       <string>Ajout</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Identifiant:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Pr√©nom:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_dateeffectation">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>210</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>140</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>310</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>80</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Adressemail:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>560</x>
         <y>210</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Dateeffectation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>220</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adressemail">
       <property name="geometry">
        <rect>
         <x>900</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>530</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri croissant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>300</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>sexe :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="sexe_le">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>300</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher un client</string>
     </attribute>
     <widget class="QTableView" name="tab_cli">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>11</y>
        <width>411</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(111, 190, 184);</string>
      </property>
     </widget>
     <widget class="QTableView" name="tab_tri">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>361</width>
        <height>271</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(111, 190, 184);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_nom">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri nom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_id">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>300</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tri_prenom">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri prenom</string>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_pb">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_ch">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>70</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf_bo">
      <property name="geometry">
       <rect>
        <x>940</x>
        <y>350</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>  PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_archiverd">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Archiver</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>280</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/raw.jpg);</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>70</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(:/img/images (2).jpg);</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>320</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
border-image: url(:/img/2516730.jpg);</string>
      </property>
      <property name="title">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_s">
      <property name="geometry">
       <rect>
        <x>980</x>
        <y>160</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>stat</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Supprimer un client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>431</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 81 11pt &quot;Rockwell Extra Bold&quot;;
background-image: url(:/img/images (1).jpg);</string>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Identifiant:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_supp">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>150</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>310</y>
         <width>211</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
color: rgb(0, 0, 0);
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Modifierunclient">
     <attribute name="title">
      <string>Modifier un client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>1071</width>
        <height>351</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/images (1).jpg);
font: 81 11pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="title">
       <string>Modifier</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Identifiant:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>190</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Pr√©nom:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Adressemail:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>230</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;Palatino Linotype&quot;;</string>
       </property>
       <property name="text">
        <string>Dateeffectation:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>130</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_prenom_2">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_adressemail_2">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>70</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_dateeffectuation_2">
       <property name="geometry">
        <rect>
         <x>910</x>
         <y>230</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>310</y>
         <width>131</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;
background-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Archivage</string>
     </attribute>
     <widget class="QTableView" name="archiveView">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>100</y>
        <width>401</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_desarchiver_do">
      <property name="geometry">
       <rect>
        <x>702</x>
        <y>70</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Desarchiver</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1259</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
